docs/en/observability/logs-threshold-alert.asciidoc

```diff
@@ -28,5 +81,144 @@ image::images/log-threshold-alert.png[Anomalies chart]
 Extend your alerts by connecting them to actions that use the following supported built-in integrations.
 
 [role="screenshot"]
-image::images/action-type-logs.png[Anomalies chart]
+image::images/action-type-logs.png[Alert action types]
+
+[discrete]
+[[es-queries]]
+==== Elasticsearch queries (advanced)
+
+When an alert check is performed we build a query based on the alert configuration provided, for the vast majority of cases it shouldn't be necessary to know what these queries are. However, it might be useful to know the structure of these queries to determine an optimal configuration, or to aid with debugging. Below is an example Elasticsearch query for the following configuration:
```

Comment:
```suggestion
When an alert check is performed, a query is built based on the alert configuration. For the vast majority of cases it shouldn't be necessary to know what these queries are. However, to determine an optimal configuration or to aid with debugging, it might be useful to see the structure of these queries. Below is an example {es} query for the following configuration:
```